style/typescript/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "arrowParens": "always",
+  "printWidth": 120,
+  "proseWrap": "always",
+  "singleQuote": true,
+  "trailingComma": "all"
```

Comment:
I've usually done "es5" for this option so it doesn't use trailing commas in function arguments. [Reference](https://prettier.io/docs/en/options.html#trailing-commas). Curious to hear your thoughts.

Reply:
👍

I made the change over in https://github.com/thoughtbot/react-suspenders/pull/1/commits/64d698b111c422390731c3528e899d67f34547bd